end-to-end-gee.Rmd

```diff
@@ -297,6 +297,20 @@ Earth Engine allows for exporting both vector and raster data to be used in an e
 ```
 [Open in Code Editor &#8599;](https://code.earthengine.google.co.in/?scriptPath=users%2Fujavalgandhi%2FEnd-to-End-GEE%3A01-Earth-Engine-Basics%2F08b_Export_(complete)){target="_blank"}
 
+Once you run this program, the *Tasks* tab will be highlighted. Click on it you will see the Exports will be read you be downloaded. Click *Run*. 
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/end_to_end_gee/export_task01.png')
+```
+
+A dialog box will be prompted to confirm the download details, after verifying click *Run* to download.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/end_to_end_gee/export_task02.png')
+```
+
+The downloaded *.tif* image can be viewed in desired solftware. 
```

Comment:
Once the Export finishes, a GeoTiff file will be added to your Google Drive in the specified folder. You can download it and use it in any GIS software.